src/blocked.c

```diff
@@ -251,6 +253,17 @@ void serveClientsBlockedOnListKey(robj *o, readyList *rl) {
                      * to also undo the POP operation. */
                     listTypePush(o,value,where);
                 }
+                const ustime_t reply_duration = ustime()-start;
+                receiver->lastcmd->microseconds += reply_duration;
+                const ustime_t total_cmd_duration = receiver->duration + reply_duration;
+                /* Log the command into the Slow log if needed. */
+                if (!(receiver->lastcmd->flags & CMD_SKIP_SLOWLOG)) {
+                    const char *latency_event = (receiver->lastcmd->flags & CMD_FAST) ?
+                                        "fast-command" : "command";
```

Comment:
this should be a new event, like "blocked-reply"

Reply:
agree. changed in the last commit